### ISSUE TITLE
Keys Detective shouldn't mark comments as extra keys

### ISSUE THREAD

**Issue by potray** [ngneat/transloco-keys-manager#56]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report 
[ ] Performance issue
[x] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

The Keys Detective marks all of our comments for translators (i.e. all entries that are `something.comment` in our `en.json` file) as extra keys. It becomes a bit cumbersome to search for real extra keys in the output of the script.

## Expected behavior

All comment entries shouldn't be marked as extra keys.

## Minimal reproduction of the problem with instructions

Have the following content in an `en.json` file:

```json
{
  "a": "this is content",
  "a.comment": "a comment about a"
}
```
Have the following content in a `.ts` file:
```typescript
import {} from '@ngneat/transloco'

/**
 * t(a)
 */
```
## What is the motivation / use case for changing the behavior?

With lots of comments, it becomes a bit hard to find all actual extra keys in the list that the Keys Detective provides.

## Environment

<pre><code>
Angular version: 9.1.12
Keys manager version: 2.2.1

For Tooling issues:
- Node version: 12.0.0 
- Platform:  Linux
</code></pre>

**Comment by shaharkazaz** [comment]
@potray released in v2.5.0, LMK if it works for you 🙂

**Comment by potray** [comment]
@shaharkazaz It does work, thanks! But I think this introduces a bug: an entry with the key `"comment"` is not processed as an extra key. I think the problem is the following two entries for `"comment"` are equivalent when reading JSON:

```json
{
  "page": "something",
  "page.comment": "something is the opposite of nothing" ⬅️ comment about the page entry, should be ignored by keys extractor
}
```
```json
{
  "page": {
    "comment": "Write your comment below" ⬅️ actual entry with a "comment" key
  }
}
```

Probably this isn't possible to implement by reading and interpreting json. The easiest solution I can think about would be to change the recommendation about writing comments to use something like a hashtag instead of a dot, so it doesn't get converted into a inner entry of the comment's key, but a genuine entry that ends with `#comment`.
